cp3pt0-deployment/common/utils.sh

```diff
@@ -233,13 +233,13 @@ function wait_for_operator() {
 function wait_for_csv() {
     local namespace=$1
     local package_name=$2
-    local condition="${OC} get subscription.operators.coreos.com -l operators.coreos.com/${package_name}.${namespace}='' -n ${namespace} -o yaml -o jsonpath='{.items[*].status.installedCSV}' | grep ^${package_name}"
+    local condition="${OC} get subscription.operators.coreos.com -l operators.coreos.com/${package_name}.${namespace}='' -n ${namespace} -o yaml -o jsonpath='{.items[*].status.installedCSV}'"
     local retries=30
     local sleep_time=10
     local total_time_mins=$(( sleep_time * retries / 60))
-    local wait_message="Waiting for operator ${package_name} CSV in namespace ${namespace} to be generated"
-    local success_message="Operator ${package_name} CSV in namespace ${namespace} is generated"
-    local error_message="Timeout after ${total_time_mins} minutes waiting for ${package_name} CSV in namespace ${namespace} to be generated"
+    local wait_message="Waiting for operator ${package_name} CSV in namespace ${namespace} to be bundled with Subscription"
+    local success_message="Operator ${package_name} CSV in namespace ${namespace} is bundled with Subscription"
+    local error_message="Timeout after ${total_time_mins} minutes waiting for ${package_name} CSV in namespace ${namespace} to be bundled with Subscription"
```

Comment:
This is used to check `.status.installedCSV`
https://github.ibm.com/ibmprivatecloud/roadmap/issues/59973